ReactWindows/ReactNative/Modules/StatusBar/IStatusBar.cs

```diff
@@ -24,7 +24,7 @@ public interface IStatusBar
         /// Hides the status bar.
         /// </summary>
         /// <returns>The asynchronous results of the operation. Use this to determine when the async call is complete</returns>
-        [System.Diagnostics.CodeAnalysis.SuppressMessage("AsyncUsage.CSharp.Reliability", "AvoidAsyncSuffix", Justification = "Reviewed.")]
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("AsyncUsage.CSharp.Reliability", "AvoidAsyncSuffix", Justification = "Intended name.")]
 #pragma warning disable AvoidAsyncSuffix 
         IAsyncAction HideAsync();
```

Comment:
Really irritating that this attribute is marked as Inherited=false :( Is there some other attribute that inherits from this one and is inheritable, @rozele ?